instrumentation/finatra-2.9/javaagent/src/latestDepTest/scala/io/opentelemetry/javaagent/instrumentation/finatra/FinatraServerLatestTest.scala

```diff
@@ -55,6 +55,7 @@ class FinatraServerLatestTest extends AbstractHttpServerTest[HttpServer] {
       override def test(endpoint: ServerEndpoint): Boolean =
         endpoint != ServerEndpoint.NOT_FOUND
     })
+    options.setResponseCodeOnNonStandardHttpMethod(400)
```

Comment:
```suggestion
    options.setResponseCodeOnNonStandardHttpMethod(404)
```

I dunno why this is needed, but I saw `NOT_FOUND` above.

Reply:
The `NOT_FOUND` you saw is probably a `ServerEndpoint` value, not a numerical status code constant.